.stylelintrc

```diff
@@ -1,28 +1,35 @@
 {
-	"extends": [ "stylelint-config-standard-scss", "@wordpress/stylelint-config/scss" ],
+	"extends": [ "@wordpress/stylelint-config/scss" ],
 	"plugins": [ "@signal-noise/stylelint-scales" ],
 	"rules": {
-		"at-rule-empty-line-before": [
-			"always",
+		"at-rule-empty-line-before": null,
+		"at-rule-no-unknown": null,
+		"comment-empty-line-before": null,
+		"font-weight-notation": null,
+		"max-line-length": null,
+		"no-descending-specificity": null,
+		"rule-empty-line-before": null,
+		"selector-class-pattern": null,
+		"value-keyword-case": null,
+		"scss/operator-no-unspaced": null,
+		"scss/selector-no-redundant-nesting-selector": null,
+		"scss/at-import-partial-extension": null,
+		"scss/no-global-function-names": null,
+		"scss/comment-no-empty": null,
+		"scss/at-extend-no-missing-placeholder": null,
+		"scss/operator-no-newline-after": null,
+		"scss/at-if-closing-brace-newline-after": null,
+		"scss/at-else-empty-line-before": null,
+		"scss/at-if-closing-brace-space-after": null,
+		"no-invalid-position-at-import-rule": null,
+		"declaration-property-unit-allowed-list": [
 			{
-				"except": [ "blockless-after-blockless" ],
-				"ignore": [ "after-comment", "first-nested" ]
-			}
-		],
-		"declaration-property-unit-allowed-list": {
-			"/^font-size$|^font$/": [ "rem" ],
-			"line-height": [ "px" ],
-			"/radius$/": [ "px" ]
-		},
-		"function-comma-space-after": "always-single-line",
-		"function-parentheses-space-inside": "always-single-line",
-		"no-descending-specificity": [
-			true,
-			{
-				"severity": "warning"
-			}
+				"/^font-size$|^font$/": [ "rem" ],
+				"line-height": [ "px" ],
+				"/radius$/": [ "px" ]
+			},
+			{ "ignore": [ "inside-function" ] }
```

Comment:
We ignore this inside of functions so that `font-size: rem(20px);` will be accepted. This is a known pattern for typography.